openstudiocore/src/project/ProblemRecord.cpp

```diff
@@ -245,12 +251,35 @@ namespace detail {
     if (!workflowRecords.empty()) {
       wrWIndex = workflowRecords[wrIndex].workflowIndex();
     }
+    OptionalRubyMeasureRecord lastRubyMeasureRecord;
+    OptionalRubyMeasure lastRubyMeasure;
     for (int i = 0, n = ivrN + wrN; i < n; ++i) {
       if (!wrWIndex || (ivrWIndex && (*ivrWIndex < *wrWIndex))) {
         // InputVariable is next
         OS_ASSERT(*ivrWIndex == wIndex); // saved and expected index into workflow match
         OS_ASSERT(ivrIndex < ivrN);      // there is an input variable record to deserialize
-        workflow.push_back(WorkflowStep(inputVariableRecords[ivrIndex].inputVariable()));
+        InputVariableRecord ivarR = inputVariableRecords[ivrIndex];

```

Comment:
This code is triggered when you open an existing project. (When a problem is deserialized from an osp.) Of course, the bulk of the code will not be triggered, because PAT doesn't support RubyContinuousVariables.